lib/process-services-cloud/src/lib/task/task-filters/components/service-task-filters-cloud.component.spec.ts

```diff
@@ -231,14 +231,11 @@ describe('ServiceTaskFiltersCloudComponent', () => {
         expect(filterClickedSpy).not.toHaveBeenCalled();
     });
 
-    it('should reset the filter when the param is undefined', async () => {
+    it('should reset the filter when the param is undefined', () => {
         const change = new SimpleChange(null, undefined, false);
         component.currentFilter = fakeGlobalServiceFilters[0];
         component.ngOnChanges({ 'filterParam': change });
 
-        fixture.detectChanges();
-        await fixture.whenStable();
```

Comment:
removing just makes the test flaky as for me

Reply:
we are not querying any HTML element so updating the fixture is redundant?

Reply:
We are not querying any html element so updating the fixture is redundant?

Reply:
Also I don't know how but keeping this code gives me a false green, removing it makes the test fail if I remove the code that resets the filter in case the filter input is undefined